UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>Oops the output below says 41? Not sure if a typo or things changed! I found it helpful to not list specific numbers or outputs in my markdown cells in case they change in the future. So you can get around this by saying e.g. "We can check the length of the maK_TWA list to confirm that we have removed the missing values:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>